app/src/main/java/com/duckduckgo/app/browser/WebDataManager.kt

```diff
@@ -95,10 +102,15 @@ class WebViewDataManager @Inject constructor(
     companion object {
         private const val WEBVIEW_DATA_DIRECTORY_NAME = "app_webview"
         private const val WEBVIEW_DEFAULT_DIRECTORY_NAME = "app_webview/Default"
+        private const val DATABASES_DIRECTORY_NAME = "databases"
 
         private val WEBVIEW_FILES_EXCLUDED_FROM_DELETION = listOf(
             "Default",
             "Cookies"
         )
+
+        private val HTTP_AUTH_FILES_TO_DELETE = listOf(
+            "http_auth.db-journal"
+        )
```

Comment:
This will always rely on the fact that http_auth.db will always use rollback journal. Which might not be always the case, as we don't currently control that db.
What do you think about on app start, setting the journal mode of the `http_auth.db` as `DELETE`? That will automatically delete the journal file after each transaction and save all this code